sdk/confidentialledger/azure-confidentialledger/setup.py

```diff
@@ -60,16 +62,24 @@
         "Programming Language :: Python :: 3.10",
         "License :: OSI Approved :: MIT License",
     ],
-    zip_safe=False,
-    packages=find_packages(exclude=[
-        "tests",
-        # Exclude packages that will be covered by PEP420 or nspkg
-        "azure"
-    ]),
     python_requires=">=3.6",
+    zip_safe=False,
+    packages=find_packages(
+        exclude=[
+            "tests",
```

Comment:
I can't leave a comment on the correct file - but the `tests` directory should not contain an `__init__.py` file.

Reply:
There's some importing happening in the tests to deduplicate code, so the `__init__.py` is necessary for the imports to resolve.